foursight_core/app_utils.py

```diff
@@ -248,6 +274,11 @@ def get_domain_and_context(cls, request_dict):
         context = '/api/' if request_dict.get('context', {}).get('path', '').startswith('/api/') else '/'
         return domain, context
 
+    def get_base_path(self, context):
+        # TODO: Have not been able to figure out where/how it is that when running
+        # in production the base path is '/api' and running in locally it is '/'.
```

Comment:
This is a detail of how `chalice` interacts with API gateway (note it is always present in the `chalice` examples), but I will say I'm having trouble tracking down any documentation on why it does this.